api-docs/cloud-dedicated/ref.yml

```diff
@@ -1576,25 +1589,43 @@ paths:
           - [Best practices for optimizing writes](/influxdb/cloud-dedicated/write-data/best-practices/optimize-writes/)
         required: true
       responses:
-        '204':
+        "201":
```

Comment:
As of right now, the decisions is to have Dedicated be:
* 204 == everything written. regardless of partial-writes config.
* 201 == partial writes, partial errors. Only occurs in partial-write mode.